src/Host/Client/Impl/Host/BrokerClient.cs

```diff
@@ -195,6 +199,13 @@ internal abstract class BrokerClient : IBrokerClient {
             return new RHost(name, callbacks, transport, cts);
         }
 
+        private async Task GetHostInformationAsync() {
```

Comment:
Don't we just get cancellation exception when we call HttpClient.CancelAllRequests ?